test.txt

```diff
@@ -0,0 +1,4253 @@
+[999D[K[999D[K[0m[7m[1m[31m FAIL [39m[22m[27m[0m [2mschema/[22m[1mresolvers.test.js[22m
```

Comment:
I presume this has been unintentionally committed?

Reply:
Whoops!